.storybook/themes/news.scss

```diff
@@ -1780,11 +1732,21 @@
 		desktop: (
 			alias: (
 				"body-font-size": var(--global-font-size-4),
-				"body-line-height": var(--global-line-height-4),
+				"body-line-height": var(--global-line-height-6),
```

Comment:
maybe body line height here may be accounting for the alert bar line height change?

Reply:
This was because the old `line-height-4` was ~`24px` and the new `line-height-6` is `1.5` which is about `24px` as well.